src/jit/gentree.cpp

```diff
@@ -394,6 +408,88 @@ bool GenTree::IsNodeProperlySized() const
 }
 #endif
 
+/*****************************************************************************
+ *
+ *  When 'NODEBASH_STATS' is enabled in "jit.h" we record all instances of
+ *  an existing GenTree node having its operator changed. This can be useful
+ *  for two (related) things - to see what is being bashed (and what isn't),
+ *  and to verify that the existing choices for what nodes are marked 'large'
+ *  are reasonable (to minimize "wasted" space).
+ *
+ *  And yes, the hash function / logic is simplistic, but it is conflict-free
```

Comment:
I didn't necessarily mean to stick this in as is (sorry I didn't mention this sooner) - I did try to find a low-overhead pre-existing hash-like thing but failed (everything I tried needed a bunch of additional #include's and who knows what else would be dragged in). Can you please recommend the right hash / hashmap / whatever for this? It's a trivial [op1,op2] -> count mapping (basically).